src/result-helpers.ts

```diff
@@ -41,3 +45,45 @@ export const normalize = <A extends unknown[]>(
     }),
   );
 };
+
+export const logServerError = (
+  error: HttpError,
+  {
+    logger,
+    request,
+    input,
+  }: {
+    logger: ActualLogger;
+    request: Request;
+    input: FlatObject | null;
+  },
+) =>
+  !error.expose &&
```

Comment:
should probably use same condition (extracted) as exposeErrorMessage does.
or probably should combine those methods.